### PR TITLE
Typos, formatting, comments in enricher

### PR DESCRIPTION
**Pull request by ocsw**
**What this PR does / why we need it**:
Fix a typo, add a couple of clarifying comments, and fix formatting of a function call.

**Which issue(s) this PR fixes** 

Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: src/gordon_gcp/plugins/enricher.py [hunk 1]
```diff
@@ -76,7 +77,7 @@ def _validate_config(self):
             errors.append(msg)
 
         if not self.config.get('dns_zone', '').endswith('.'):
-            msg = 'A dns zone must be a FQDN and end with the root zone (".").'
+            msg = 'A dns zone must be an FQDN and end with the root zone (".").'
             errors.append(msg)
```
Comment (descent3me) [inline src/gordon_gcp/plugins/enricher.py]: Actually, it's unrelated to typos, but I don't see the point in fixing typos for something that should change. If you wanted to make a new commit to just drop this whole if-block, and maybe reword the msg in the one above, that would be better, so that it would match the format expected by the janitor https://github.com/spotify/gordon-janitor-gcp/blob/develop/src/gordon_janitor_gcp/plugins/authority.py#L92

File: src/gordon_gcp/plugins/enricher.py [hunk 2]
```diff
@@ -89,6 +90,7 @@ def _validate_config(self):
             self.config['retries'] = 5
 
     def _init_auth(self):
+        # req keys: scopes, keyfile
```
Comment (descent3me) [inline src/gordon_gcp/plugins/enricher.py]: I'm not sure this makes sense to include here, since the validation should already have happened at this point and we know `keyfile` is in the config. `scopes` is not required https://github.com/spotify/gordon-gcp/blob/develop/src/gordon_gcp/clients/auth.py#L80

File: src/gordon_gcp/plugins/enricher.py (hunk 1 above)
Comment (ocsw) [reply src/gordon_gcp/plugins/enricher.py]: If I understand you correctly, the janitor one seems like it's missing the FQDN test.  Should we change that side?

Comment (descent3me) [reply src/gordon_gcp/plugins/enricher.py]: It was intentionally removed because none of the DNS provider APIs that I checked (Google, DYN, NS1, Route53)  expect a zone identifier in that format

Comment (descent3me) [reply src/gordon_gcp/plugins/enricher.py]: I retract the above comment.  I now (again) think it might be a good idea to use the absolute zone name, regardless of what the API's expect, since I think it makes more sense to us logically while reasoning about the system.  I will update `gordon-janitor-gcp`

**Comment by descent3me** [comment]
I would update the commit message with a verb like "Fix", but otherwise +1
